client/src/main/kotlin/no/nav/su/se/bakover/client/ProdClientsBuilder.kt

```diff
@@ -69,7 +72,8 @@ data class ProdClientsBuilder(internal val jmsContext: JMSContext) : ClientsBuil
                         jmsContext = jmsContext
                     )
                 }
-            )
+            ),
+            adGraphApiClient = AdGraphApiClient(oAuth),
```

Comment:
Kunne kanskje bare vært en innbakt del av den eksisterende "AzureClient" med et nytt interface f.eks? Eller er det viktig for oss å "vite" om hvilket api vi benytter?

Reply:
Godt innspill! Nei, det er egentlig ganske unødvendig å måtte forholde seg til forskjellige ting. Jeg tar og flytter det inn dit :)

Reply:
Jeg synes det bør være en egen klient, siden den bruker en annen protokoll. Men om den blir eksponert i samme interface kan jeg være åpen for (da må vi ha en wrapper i så fall). Vi vil kanskje alltid gjøre begge deler sammen? Da bør de vel smeltes sammen i en data class

Jeg savner uansett et interface som har et litt mindre implementasjonsspesifikt navn som ligger i domain.

Reply:
Hmmm, etter å ha tenkt litt mer på det så er jeg litt mer ambivalent.

Dette hører på en måte mer hjemme i noe ala `PersonClient`; bare at vi her henter informasjon om innlogget bruker og ikke på et fnr/navident som man sender inn 🤔

Reply:
Enig, jeg synes og den passer inn i personClient, og den har vel allerede wrapper-kode for dette siden den integrerer mot fler tjenester.